client/themes/zenburn.css

```diff
@@ -59,7 +59,7 @@ body {
 }
 
 #footer {
-	background: #33332f;
+	background: #333;
```

Comment:
Pretty sure this was a typo, because this is 1 digit off, so probably a <kbd>Ctrl</kbd><kbd>Down</kbd> in a text editor or something.

Junction between footer and input margin, before:

<img width="34" alt="screen shot 2017-07-04 at 13 37 25" src="https://user-images.githubusercontent.com/113730/27839365-a6aad814-60be-11e7-88fa-bb2fd6e3f6dd.png">